services_and_applications/slate/access_olcf_resources/mount_fs.rst

```diff
@@ -10,8 +10,10 @@ to get the necessary configuration injected into the container at runtime.
   :header: "Cluster", "Annotation", "Value", "Mounts"
   :widths: 5, 8, 5, 25
 
-  "Marble", "ccs.ornl.gov/fs", "olcf", "/ccs/sw, /ccs/home, /ccs/proj, /gpfs/alpine"
+  "Marble", "ccs.ornl.gov/fs", "olcf", "/ccs/sw, /ccs/home, /ccs/sys, /ccs/proj, /gpfs/alpine"
+  "Marble", "ccs.ornl.gov/fs", "olcf-nfs", "/ccs/sw, /ccs/home, /ccs/sys, /ccs/proj"
   "Onyx", "ccs.ornl.gov/fs", "ccsopen", "/ccsopen/sw, /ccsopen/home, /ccsopen/proj, /gpfs/wolf"
+  "Onyx", "ccs.ornl.gov/fs", "open-nfs", "/ccsopen/sw, /ccsopen/home, /ccsopen/proj"
```

Comment:
should this match "ccsopen" as "ccsopen-nfs"?

Reply:
Yes. I'll add that to the opa rules and then update here before merging. Do you think we should keep the "open-nfs" annotation as an undocumented option in case people are already using it?

Reply:
good thought but I know no one is using it right now so we can drop it

Reply:
Alright, updated the opa rules and changed it here